### ISSUE TITLE
bare metal

### ISSUE THREAD

**Issue by sergundo** [LuckfoxTECH/luckfox-pico#172]
I didn't find any information on the wiki about how to use luckfox pico for bare metal programming.

**Comment by luckfox-eng33** [comment]
Yes, in general Rockchip's chips don't provide information about how to program bare metal

**Comment by redstonee** [comment]
Xboot seems to have implemented this [here](https://github.com/xboot/xboot/tree/master/src/arch/arm32/mach-rv1106)
